### PR TITLE
Integrate simulation to DFL

### PR DESCRIPTION
**Pull request by lecourtoisflo**
Use jobentry instead of writing a job file
Create a simulation and launche it. For now, since all inputs files are not
generated, this code is commented

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -78,10 +78,24 @@ set_optimization()
...
 list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}")
+list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}/cmake")
 list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}/share")
-find_package(Dynawo 1.3.0 REQUIRED)
-#libXML
 list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}/share/cmake")
+list(APPEND CMAKE_MODULE_PATH "${DYNAWO_HOME}/cmake")
+list(APPEND CMAKE_MODULE_PATH "${DYNAWO_HOME}/share/cmake")
+
+find_package(Dynawo 1.3.0 REQUIRED)
```
Comment (gautierbureau) [inline CMakeLists.txt:87]: dynawo should be place after its dependencies

File: sources/Context.h [hunk 2]
```diff
@@ -32,16 +33,26 @@ namespace dfl {
...
+  /**
+   * @brief Context definition
+   */
+  struct ContextDef {
+    std::string networkFilepath;  ///< network filepath
+    std::string configFilepath;   ///< configuration filepath
+    std::string dynawLogLevel;    ///< string representation of the dynawo log level
+    std::string parFileDir;       ///< parameter file directory
+    std::string dynawoResDir;     ///< DYNAWO resources
+    std::string locale;           ///< lcoalization
```
Comment (gautierbureau) [inline sources/Context.h:46]: typo

**Review by gautierbureau — CHANGES_REQUESTED** [review]

File: sources/Context.h (hunk 2 above)
Comment (lecourtoisflo) [reply sources/Context.h:46]: OK

File: CMakeLists.txt (hunk 1 above)
Comment (lecourtoisflo) [reply CMakeLists.txt:87]: OK

**Review by rosiereflo — APPROVED** [review]

**Review by gautierbureau — APPROVED** [review]

**Review by PerninGuillaume — APPROVED** [review]

**Pull request merged**